packages/donor/src/components/DonationInfo/DonationInfo.tsx

```diff
@@ -1,24 +1,27 @@
-import styles from "./DonationToBookInfo.module.scss";
+import styles from "./DonationInfo.module.scss";
 import Calendar from "../../assets/images/AppointmentCalendar.svg";
 import { DateUtils, Hospital, LocaleUtils } from "@zm-blood-components/common";
-import React from "react";
 
 export interface DonationToBookInfoProps {
   donationStartTimeMillis: number;
   hospital: Hospital;
+  isPreviousAppointmentInfo: boolean;
```

Comment:
1. I think you mean existing, not previous.
2. This is a component, and a field like `isPreviousAppointmentInfo` is very coupled with the domain logic. Please try a name that does not leak this logic inside. Either you rename the field, or take the entire title as a string parameter.

Reply:
Done